tests/providers/apache/hive/transfers/test_mysql_to_hive.py

```diff
@@ -277,7 +277,7 @@ def test_mysql_to_hive_verify_csv_special_char(self, spy_on_hive):
                     )
                     conn.commit()
 
-            import unicodecsv as csv
+            import csv
```

Comment:
We can move this import to the top of the file

Reply:
did this